modules/service/src/main/scala/lucuma/odb/graphql/input/WhereOptionEq.scala

```diff
@@ -25,8 +26,8 @@ object WhereOptionEq {
           (isNull, EQ, NEQ, IN, NIN) =>
             and(List(
               isNull.map(IsNull(path, _)),
-              EQ.map(a => Eql(path, Const(a))),
-              NEQ.map(a => NEql(path, Const(a))),
+              EQ.map(a => Eql(path, Const(a.some))),
+              NEQ.map(a => NEql(path, Const(a.some))),
               IN.map(as => In(path, as)),
```

Comment:
Should that be `as.map(_.some)` for `IN` and `NIN`?  Maybe add a test or change the `EQ` test to `IN`?

Reply:
I'll add a test for `IN`